packages/react/src/app-layout/AppLayoutSidebarNav.tsx

```diff
@@ -337,6 +337,20 @@ function AppLayoutSidebarNavItemInner({
 					},
 
 					...focusStyles,
+					':focus-visible': {
+						// Elevate focused item above next sibling
+						zIndex: tokens.zIndex.elevated,
+						// Bring the focus ring in against the element so that it remains on screen and sibling element borders don't show in the offset
+						outlineOffset: `-${packs.outline.outlineOffset}px`,
```

Comment:
Honestly, I'd just make this `0`

Reply:
I don't think that works with sibling borders but I'll check.

- [ ] check sibling borders with `outlineOffset: 0`

Reply:
Oh right, it hides the sides of the outline. I think this negative offset is better because nothing is obscured and you get the full outline.